info.xml

```diff
@@ -16,12 +16,15 @@
     <url desc="Licensing">http://www.gnu.org/licenses/agpl-3.0.html</url>
   </urls>
   <releaseDate>2019-03-26</releaseDate>
```

Comment:
Is this date valid? I would think it would be a more recent date/

Reply:
I did not update the date yet, this will be updated when we make a release.